pythonforandroid/recipe.py

```diff
@@ -783,8 +783,13 @@ def get_recipe_env(self, arch=None, with_flags_in_cc=True):
             hppath.append(join(dirname(self.hostpython_location), 'Lib'))
             hppath.append(join(hppath[0], 'site-packages'))
             builddir = join(dirname(self.hostpython_location), 'build')
-            hppath += [join(builddir, d) for d in listdir(builddir)
-                       if isdir(join(builddir, d))]
+            try:
+                listed = listdir(builddir)
+                hppath += [join(builddir, d) for d in listed
+                           if isdir(join(builddir, d))]
+            except OSError as err:
+                debug('%s', err)
```

Comment:
Did you have a particular issue that you're catching with this? I'm a little wary of swallowing the error with a debug message if it could be something important, but I haven't checked if that's likely to be a problem.